Gems/WhiteBox/Code/CMakeLists.txt

```
@@ -44,6 +43,9 @@ if(NOT PAL_TRAIT_WHITEBOX_SUPPORTED)
             BUILD_DEPENDENCIES
                 PRIVATE
                     AZ::AzCore
+            RUNTIME_DEPENDENCIES
+                PRIVATE
+                    Gem::EditorPythonBindings.Editor
```

`RUNTIME_DEPENDENCIES` doesn't have use scope for target propagation.
`PRIVATE` needs to be removed
```suggestion
            RUNTIME_DEPENDENCIES
                Gem::EditorPythonBindings.Editor
```